src/main/java/racingcar/domain/Position.java

```diff
@@ -17,7 +15,7 @@ private void validatePosition(int position) {
 
     private void validatePositionIsNotNegative(int position) {
         if (position < 0) {
-            throw new PositionInvalidException();
+            throw new IllegalStateException("위치는 음수일 수 없습니다.");
```

Comment:
```suggestion
            throw new IllegalStateException("위치는 음수일 수 없습니다. 입력된 포지션 : " + position);
```
어떤 값이 입력되어 오류가 발생했는지 로깅이나 오류 메시지로 표시하는 습관이 들어있으면 정말 큰 도움이 됩니다 :)

Reply:
오우 말씀해주신 친절한 코드군요..! 습관을 들여보겠습니다 😄